### PR TITLE
Change getCapabilities output

### PR DESCRIPTION
**Pull request by shucon**
Partial-PR: #8020

**Thanks for contributing to Selenium!**
**A PR well described will help maintainers to quickly review and merge it**

Before submitting your PR, please check our [contributing](https://github.com/SeleniumHQ/selenium/blob/master/CONTRIBUTING.md) guidelines.
Avoid large PRs, help reviewers by making them as simple and short as possible.

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist

- [ ] I have read the [contributing](https://github.com/SeleniumHQ/selenium/blob/master/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by shucon** [comment]
@AutomatedTester

File: java/server/src/org/openqa/selenium/grid/graphql/Node.java [hunk 1]
```diff
@@ -59,6 +58,22 @@ public int getMaxSession() {
     return maxSession;
   }
 
+  public StringBuilder getCapabilities() {
+    StringBuilder str = new StringBuilder("[");
```
Comment (AutomatedTester) [inline java/server/src/org/openqa/selenium/grid/graphql/Node.java]: We could simplify this greatly by having this method return a Map of what we want it to hold and the call `JSON.toJson` on our new map

Comment (shucon) [reply java/server/src/org/openqa/selenium/grid/graphql/Node.java]: I'm a bit confused, what would be the key and value of map in that case?

File: java/server/src/org/openqa/selenium/grid/graphql/Node.java [hunk 2]
```diff
@@ -59,6 +58,22 @@ public int getMaxSession() {
...
+
+    for (Map.Entry<Capabilities, Integer> entry : capabilities.entrySet()) {
+      Capabilities k = entry.getKey();
```
Comment (AutomatedTester) [inline java/server/src/org/openqa/selenium/grid/graphql/Node.java]: Do something like the following.
```java 
List<Map> toReturn = new List<>();
...
//now inside your loop
Map<String, Object> details  = new Map<>();
details.put("browserName", entry.getKey().getBrowserName());
details.put("slot", entry.getValue());
toReturn.add(details);
```
And then have `JSON.toJSON` be called on the return from your new method which will return a `List<Map>` now.

**Review by AutomatedTester — CHANGES_REQUESTED** [review]
Setting request changes from previous comments

**Review by AutomatedTester — APPROVED** [review]

**Pull request merged**